assets/style/style.css

```diff
@@ -2277,6 +2277,10 @@ a {
   font-size: 17px;
 }
 
+.dcWallet-balances .dcWallet-pivxImmatureBalance {
+    font-size: 17px;
+}
```

Comment:
You can add this directly to the .vue component with a `<style>` tag (You can then remove the `dcWallet` prefix, I'll eventually do this for all component specific css